### DISCUSSION TITLE
SSL Certificate - I need a modification

### DISCUSSION THREAD

**Discussion by devilismro** [vercel/community#4649] [Help]
### Summary

I have a big issue. Before Vercel, I used to host my website on another server. And I still have some subdomains "active" on that server. Like webmail.mydomain.io, etc. For whatever reason, my wildcard SSL is addresed to *.mydomain.io and mydomain.io. What I want is that only mydomain.io and www.mydomain.io to have SSL, and to delete that *.mydomain.io from the certificate. 
But, my issue is that the present certficate for mydomain.io includes the *.mydomain.io. If I delete this certificate, will I be able, from Vercel, to issue another one valid only for mydomain.io? 
I tried to modify using Vercel CLI using : vercel domains rm *.mydomain.io , and I got an error : "Domain not found". 
Thank you

### Steps to Reproduce

PS C:\Users\iulian\Desktop\presentationn> vercel domains rm *.mydomain.io
Vercel CLI 28.20.0
Error: Domain not found by "*.mydomain.io" under i***-gmailcom
> Run `vercel domains ls` to see your domains.
PS C:\Users\iulian\Desktop\presentation> vercel certs ls
Vercel CLI 28.20.0
> Certificates found under ****-gmailcom [650ms]
id                             cns                  expiration  renew  age
 cert_KDlmXvz7xO0cm5oc8jZtfIqz  - *mydomain.io            in 50d   yes   40d
                                 - mydomain.io
  cert_aH4DoTcdAzg8fehi8c5XqXLG  www.mydomain.io            in 47d   yes   43d
  cert_p6cAcflhPXjoCwi6rW3VGEe7  mydomain.io                in 47d   yes   43d

**Comment by swarnava** [comment]
Hi there,

We can only issue certificate that is hosted in Vercel. Is mydomain.io hosted in Vercel and linked to a project?

**Reply by devilismro** [reply]
Hy. Thanks for your reply. My domain is hosted on Vercel, of course and linked to my project. My domain's name is myvolt.io . And, like I said, my wildcard domain certificate is issued for *.myvolt.io and myvolt.io. But I wanna remove the *.myvolt.io. That's it. To have only for www.myvolt.io and myvolt.io. Can you help me?
